script/proof

```diff
@@ -32,4 +32,4 @@ bundle exec jekyll build -s $SOURCE -d $DESTINATION --trace
 
 # 3.
 msg "Proofing..."
-time bundle exec htmlproofer ./$DESTINATION --url-ignore $INGORE_HREFS $@
+time bundle exec htmlproofer ./$DESTINATION --allow-hash-href --disable-external --url-ignore $IGNORE_HREFS $@
```

Comment:
Right, let's deal with this in another PR, the primary goal was to fix the current errors.
It should be less painful to run the script when #5771 will be done anyway.